scipy/signal/filter_design.py

```diff
@@ -4385,7 +4386,7 @@ def _bessel_poly(n, reverse=False):
     out = []
     for k in range(n + 1):
         num = _falling_factorial(2*n - k, n)
-        den = 2**(n - k) * factorial(k, exact=True)
```

Comment:
this seems to be using integer before, I'm not sure the change is appropriate here.

Reply:
Changed to `math.factorial` instead. Again much faster than using `scipy.special`'s ufuncs for a scalar.